src/controllers/assignUsersToManager.js

```diff
@@ -23,10 +23,11 @@ const assignUsersToManager = async (req, res) => {
 
     const notification = await createNotification(newNotificantion);
 
-    pusher.trigger('bare-foot-normad', 'notification', { notification });
+    // pusher.trigger('bare-foot-normad', 'notification', { notification });
```

Comment:
@Noel2002 I'd like to know any reason why you've commented this line yet this PR is about sending & receiving push notifications.

Reply:
**Pusher** was the one used by Cohort 2 to send in-app notifications. But, for me, I used **web-push** to perform that task. That's why I commented it because it was no longer useful. I may have forgotten to remove it.